plugin/src/test/groovy/org/gradle/testretry/testframework/SpockFuncTest.groovy

```diff
@@ -853,7 +840,7 @@ class SpockFuncTest extends AbstractFrameworkFuncTest {
         return """
             dependencies {
                 implementation "org.codehaus.groovy:groovy-all:2.5.8"
-                testImplementation "org.spockframework:spock-core:1.3-groovy-2.5"
+                implementation "org.spockframework:spock-core:1.3-groovy-2.5"
```

Comment:
build gets confused otherwise

Reply:
❓ How so?

Reply:
because the parent is in main, i.e. https://github.com/gradle/test-retry-gradle-plugin/blob/3578f5d29412e580e9b5d272ef2fc34b5f9161e0/plugin/src/test/groovy/org/gradle/testretry/testframework/SpockFuncTest.groovy#L673 - should I move that to a test folder as well?

Reply:
So just for this one test? In that case, I think we should parameterize it or add an implementation dependency just for that test.

Reply:
I'll add a replace for this one test to change test dependencies to prod ones - at least this way other tests won't be affected

Reply:
Done